### PR TITLE
Add symbol to list of primitives

### PR DESCRIPTION
**Pull request by arnav-aggarwal**

### CODE REVIEW

**Comment by arnav-aggarwal** [comment]
Would you like me to make that change myself? I ask because this PR is not marked as "Changes requested".

**Comment by ljharb** [comment]
Yes, that'd be great :-) I'll hold off merging until then.

**Comment by arnav-aggarwal** [comment]
Done. Let me know if you'd like something changed.

File: README.md [hunk 1]
```diff
@@ -78,8 +79,11 @@ Other Style Guides
     console.log(foo, bar); // => 1, 9
     ```
 
-  <a name="types--complex"></a><a name="1.2"></a>
-  - [1.2](#types--complex)  **Complex**: When you access a complex type you work on a reference to its value.
+  <a name="types--symbols"></a><a name="1.2"></a>
+  - [1.2](#types--symbols)  **Symbols**: Symbols cannot be faithfully polyfilled, so they should not be used when targeting browsers/environments that don't support them natively.
+
+  <a name="types--complex"></a><a name="1.3"></a>
```
Comment (ljharb) [inline README.md]: The legacy numbered links need to stay; iow, the "complex" section needs to forever be linked under `1.2`.

File: README.md [hunk 2]
```diff
@@ -78,8 +79,11 @@ Other Style Guides
     console.log(foo, bar); // => 1, 9
     ```
 
-  <a name="types--complex"></a><a name="1.2"></a>
-  - [1.2](#types--complex)  **Complex**: When you access a complex type you work on a reference to its value.
+  <a name="types--symbols"></a><a name="1.2"></a>
```
Comment (ljharb) [inline README.md]: new sections don't get numbered links; just the named ones.

**Comment by arnav-aggarwal** [comment]
Updated, I think the way you want. I wasn't sure what you meant by your note "new sections don't get numbered links; just the named ones", so I did my best.

**Pull request merged**